Gateway/Request/APMBuilder.php

```diff
@@ -9,13 +9,14 @@
 use Omise\Payment\Model\Config\Internetbanking;
 use Omise\Payment\Model\Config\Tesco;
 use Omise\Payment\Model\Config\Truemoney;
+use Omise\Payment\Model\Config\Conveniencestore;
 use Omise\Payment\Observer\InstallmentDataAssignObserver;
 use Omise\Payment\Observer\InternetbankingDataAssignObserver;
 use Omise\Payment\Observer\TruemoneyDataAssignObserver;
+use Omise\Payment\Observer\ConveniencestoreDataAssignObserver;
```

Comment:
Just very minor but I think you want to keep it in alphabetically order (guessing from the current order)

Reply:
nooo... it was mixed between the time of adding, and randomness, internet banking/true money/convenience store/ that was the time of adding.

Reply:
I thought we re-ordered it to be in alphabetically order when we introduced this APMBuilder class 😂  (as Alipay comes first, then follows by Installment and InternetBanking).
However, it's just super minor. I'll mark it as resolve : )